### PR TITLE
key equations tables alignement hs physics

### PR DESCRIPTION
**Pull request by ajluk**
Issue:  #1911

### CODE REVIEW

File: styles/templates/template1/parts/_eoc-settings.scss [hunk 1]
```diff
@@ -528,11 +528,13 @@
...
         ),
-        TableContainer: (
+        KeyEquationsTableContainer: (
             margin-bottom: v-grid(2),
+            margin-left: 0,
+            margin-right: 0,
         ),
         TableData: (
-            padding: v-grid(1) h-grid(1), 
+            padding: v-grid(.5) h-grid(.5),
```
Comment (Stackblocks) [inline styles/templates/template1/parts/_eoc-settings.scss]: Should this use `table-v-grid(1)` and `table-h-grid(1)` so it is aligned with the other table spacing constructs?

File: styles/designs/carnival/parts/_eoc-components.scss [hunk 2]
```diff
@@ -286,3 +286,12 @@ $Exercise_Table_Row: (
         padding: enum('ValueSet:::OPTIONAL'),
     )
 );
+$Key_Equations_Table_Container: (
+    _name: "KeyEquationsTableContainer",
+    _subselector: " > .os-table",
+    _properties:(
+        margin-bottom: enum('ValueSet:::REQUIRED'),
+        margin-left: enum('ValueSet:::OPTIONAL'),
+        margin-right: enum('ValueSet:::OPTIONAL'),
```
Comment (wilkus27) [inline styles/designs/carnival/parts/_eoc-components.scss]: Is there a reason why those properties are not hard coded? I think that Key Equation always has to have such margins, but maybe I miss something.

Comment (ajluk) [reply styles/designs/carnival/parts/_eoc-components.scss]: @wilkus27 Yes, you're right. I have replaced them as hard coded. The code is ready for review.

File: styles/templates/template1/parts/_eoc-settings.scss (hunk 1 above)
Comment (ajluk) [reply styles/templates/template1/parts/_eoc-settings.scss]: @Stackblocks you're right I replaced the functions with table-grid ones.

File: styles/output/hs-physics-pdf.css [hunk 3]
```diff
@@ -1559,14 +1559,16 @@ iframe {
...
-  margin-bottom: 3rem; }
+.os-eoc.os-key-equations-container > .key-equations > .os-table {
+  margin-bottom: 1.5rem;
+  margin-left: 0;
+  margin-right: 0; }
 
-.os-eoc.os-key-equations-container > .key-equations > .unnumbered.unstyled > thead > tr > th {
-  padding: 1.5rem 8px; }
+.os-eoc.os-key-equations-container > .key-equations > .os-table > .unnumbered.unstyled > thead > tr > th {
+  padding: 0.75rem 0.75rem; }
```
Comment (wilkus27) [inline styles/output/hs-physics-pdf.css]: @Stackblocks I wasn't sure why after Ania applied `table-v-grid` and `table-h-grid` function units changed from px to rem. I checked in the code and both functions use `v-grid`:

image

Is it intentional?

**Comment by brittweinstein** [comment]
@ajluk we can merge this one

File: styles/output/hs-physics-pdf.css (hunk 3 above)
Comment (Stackblocks) [reply styles/output/hs-physics-pdf.css]: image

@wilkus27 the `table-v/h-grids` both use the older `half-v-grid` and are simply a vertical and horizontal pair of aliases, so this is expected behavior.

Comment (wilkus27) [reply styles/output/hs-physics-pdf.css]: Ok :+1: Thank you @Stackblocks for confirmation :relaxed:

**Pull request merged**